scripts/test/SceneBuilder.ps1

```diff
@@ -0,0 +1,300 @@
+   $UnityPath = ${Env:${{ parameters.UnityVersion }}}
+   # Find unity.exe as Start-UnityEditor currently doesn't support arbitrary parameters
+   $editor = Get-ChildItem $UnityPath -Filter 'Unity.exe' -Recurse | Select-Object -First 1 -ExpandProperty FullName
+   
+   # The build output goes to a unique combination of Platform + Arch + Scene to ensure that
+   # each build will have a fresh destination folder.
+   $outDir = "$(Build.ArtifactStagingDirectory)\build\${{ parameters.UnityVersion }}\${{ parameters.Platform }}_${{ parameters.Arch }}_${{ parameters.Scene }}"
+   $logFile = New-Item -Path "$outDir\build\build.log" -ItemType File -Force
+   $logDirectory = "$outDir\logs"
+   $projectPath = $(Get-Location)
+   $sceneList = ''
+   if ("${{ parameters.Scene }}" -eq "HandInteractionExamples")
```

Comment:
One thing to be aware of is this expansion of short name -> long name has long term maintenance cost (i.e. if things get moved, there's nothing that will alert us to update this mapping). It can be easier to take in a full path as a result - it makes usage of it harder but also makes maintenance easier.